sms/twilio-php/README.md

```diff
@@ -0,0 +1,3 @@
+# Twilio SMS integration
+
+Download PHP Helper Library - http://github.com/twilio/twilio-php
```

Comment:
Might be good to tell users how the download should be unpacked (i.e., "Make sure that Twilio.php is at `twilio-php/Services/Twilio.php`.)  I haven't unpacked the library yet; if it contains the `twilio-php` directory, maybe say "Delete this folder and replace it with...".  Specify if root folder should be renamed.